scripts/scr_battle_roster/scr_battle_roster.gml

```diff
@@ -121,87 +131,87 @@ function scr_battle_roster(_battle_loci, _loci_specific, _is_planet) {
 
 	                    if (obj_ncombat.battle_special="space_hulk") then obj_ncombat.player_starting_dudes+=1;
 
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,12]){col=obj_controller.bat_scout_column;obj_ncombat.scouts+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,8]){col=obj_controller.bat_tactical_column;obj_ncombat.tacticals+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,3]){col=obj_controller.bat_veteran_column;obj_ncombat.veterans+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,9]){col=obj_controller.bat_devastator_column;obj_ncombat.devastators+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,10]){col=obj_controller.bat_assault_column;obj_ncombat.assaults+=1;}
-	                    if (obj_ini.role[cooh,va]=obj_ini.role[100,17]){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
-	                    if (obj_ini.role[cooh,va]=string(obj_ini.role[100,17])+" Aspirant"){col=obj_controller.bat_librarian_column;obj_ncombat.librarians+=1;moov=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,12]){col=obj_controller.bat_scout_column;obj_ncombat.scouts+=1;}
+	                    if (deploying_unit.role[cooh,va]=deploying_unit.role[100,8]){col=obj_controller.bat_tactical_column;obj_ncombat.tacticals+=1;}
```

Comment:
cooh? va?

Reply:
Just co and v, but modificed in case meat=true. If I ever get around thinking how to refactor all the combat stuff (with all the Alarms and stuff it's a Go To/Break nightmare even when debugging), will probably get rid of it.